pkg/cmd/admin/router/router.go

```diff
@@ -789,7 +790,18 @@ func RunCmdRouter(f *clientcmd.Factory, cmd *cobra.Command, out, errout io.Write
 		return defaultOutputErr
 	}
 
-	if errs := cfg.Action.WithMessage(fmt.Sprintf("Creating router %s", cfg.Name), "created").Run(list, namespace); len(errs) > 0 {
+	levelFilter := func(e error) string {
+		if match, _ := regexp.MatchString("serviceaccounts|rolebinding.*already exists", e.Error()); match {
```

Comment:
could be anything that you'd consider less fragile, a check against IsAlreadyExists and then looking for SA for example.